TheWarWithin/Priorities/RogueOutlaw.simc

```diff
@@ -75,7 +75,7 @@ actions.cds+=/call_action_list,name=stealth_cds,if=!stealthed.all
 actions.cds+=/thistle_tea,if=!buff.thistle_tea.up&(energy.base_deficit>=150|fight_remains<charges*6)
 # Use Blade Rush at minimal energy outside of stealth
 actions.cds+=/blade_rush,if=energy.base_time_to_max>4&!stealthed.all
-actions.cds+=/potion,if=buff.bloodlust.react|fight_remains<30|buff.adrenaline_rush.up
+actions.cds+=/potion,if=buff.bloodlust.up|fight_remains<30|buff.adrenaline_rush.up
```

Comment:
Let's do `boss&fight_remains<30` so we're not blowing potion CD on every short-lived pack.